File: Math/412. Fizz Buzz/412. Fizz Buzz.py
from typing import List
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        #Loop for filling the String's List:
        for num in range(1, n + 1): #We start counting from 1 and have to include number n
            
            if(num % 3 == 0 and num % 5 == 0):
                answer.append("FizzBuzz")
            elif(num % 3 == 0):
                answer.append("Fizz")
            elif(num % 5 == 0):
                answer.append("Buzz")
            else:
                answer.append(str(num))
        return answer
            

File: Math/412. Fizz Buzz/412.Fizz Buzz.ts
function fizzBuzz(n: number): string[] {
    let answer:string[] = [];
    for(let i : number = 1 ; i < n+1; i++){
        if(i % 3 == 0 && i % 5 == 0){
            answer.push("FizzBuzz");
        }
        else if(i % 3 == 0)
            answer.push("Fizz");
        else if(i % 5 == 0)
            answer.push("Buzz");
        else{
            answer.push(String(i)); 
        }
    }
    return answer;
    
};